production/helm/loki/templates/monitoring/servicemonitor.yaml

```diff
@@ -1,5 +1,6 @@
----
+{{- if .Values.monitoring.serviceMonitor.enabled }}
```

Comment:
nit: if we move that down a line, we can just say `{{ if .enabled }}` which might read a tad nicer, wdyt?

Reply:
Happy to make the change, although it wouldn't be consistent with the other monitoring resources? Thoughts?

https://github.com/grafana/loki/blob/75648288ac1dbf2b39a81f47dac0def381399b33/production/helm/loki/templates/monitoring/grafana-agent.yaml#L1
https://github.com/grafana/loki/blob/75648288ac1dbf2b39a81f47dac0def381399b33/production/helm/loki/templates/monitoring/prometheus-rules.yaml#L1

Reply:
fixed: https://github.com/grafana/loki/pull/7096

Reply:
Nice one, I was happy to include those changes but wanted the go ahead first, so thanks for that! 😅  

I've updated the conditional to match your PR, once yours is merged I'll bump the chart versions here too.